roles/DCOS.bootstrap/tasks/main.yml

```diff
@@ -70,9 +70,41 @@
      map(attribute='dcos_installation') | map(attribute='version') | list |
      union([dcos['version_to_upgrade_from'] | default(dcos['version']) ]) | unique }}
 
-- name: Ceate install directory/genconf
+- name: Create install directory/genconf
   file: path={{ download_path }}/genconf state=directory mode=0755
 
+- name: Create bootstrap CA directory
+  file: path={{ download_path }}/genconf/ca state=directory mode=0700
+
+- name: "Generate PSK for exhibitor bootstrap"
```

Comment:
this changes on every run. It will never reach a ok state.

Reply:
@fatz  Given multiple bootstrap nodes, this could be a problem... it this a common scenario, where bootstrap_url is a VIP?

I had initially generated the PSK on localhost, though this proved difficult on computers not running linux.

Reply:
its not about bootstrap redundancy. This is about multi region clusters having a bootstrap node in each region. Its neither a official feature or broadly used but it was possible in the past as the bootstrap node was stateless